File: binaryTree/python/Binary Search Tree/post-order-traversal.py
class TreeNode:
    def __init__(self, val=0, left=None, right=None):
        self.val = val
        self.left = left
        self.right = right

class Solution:
    def postorderTraversal(self, root: TreeNode):
        result = []
        if root is not None:
            result = self.postorderTraversal(root.left)
            result += self.postorderTraversal(root.right)
            result.append(root.val)
        
        return result
        

File: Hackerrank Solutions/c++/Trees/postOrder Traversal/postOrder Traversal.cpp

class Node {
    public:
        int data;
        Node *left;
        Node *right;
        Node(int d) {
            data = d;
            left = NULL;
            right = NULL;
        }
};

*/


    void postOrder(Node *root) {
if(root==NULL)
return ;
{
    postOrder(root->left);
    postOrder(root->right);
    cout<<root->data <<" ";
    
}
    }

//~YashPassi

